definitions/variable/macro-economy/price.yaml

```diff
@@ -0,0 +1,26 @@
+- Price|Carbon:
```

Comment:
I am wondering - is it useful to have a "pure" variable without explicit region aggregation? 
I fear this could lead to constant need for questions back to the modelers "how did you aggregate this", and I am not sure we gain anything? 

also, what would the scenario explorer do for the automatic aggregation - doesn't it need a specification which of the following variants is applied when reading in "Price|Carbon"? 

Maybe I overlook something, but otherwise I would propose to drop this general term and only have the three explicit versions below.

Reply:
Ah, so teams upload with "Price|Carbon", and then in the explorer one will see "Price|Carbon [Mean]" (or whatever was decided as aggregation mechanism by the project lead?